IPs/RSKIP176.md

```diff
@@ -0,0 +1,145 @@
+# Fast BTC Bridge
+|RSKIP          |           |
+| :------------ |:-------------|
+|**Title**      |Fast BTC Bridge |
+|**Created**    |15-SEP-20 |
+|**Author**     |GM |
+|**Purpose**    |USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+
+## Previous concepts
+
+#####Derived Federation
+The derived federation is the result of a process that will be carried out by creating a 32 bytes hash built from some provided derivation arguments and the active federation redeem script. The derivation arguments are values used to close the agreement between the Liquidity Provider and the user.
```

Comment:
Suggested rewording:
The derived Powpeg address results from a transformation of the active Powpeg redeem script and a 32 bytes hash of some arguments related to that specific peg-in operation .The arguments are agreed by a Liquidity Provider and a user who wishes to peg-in.